packages/storage-plus/src/indexed_map.rs

```diff
@@ -136,6 +136,11 @@ where
     pub fn sub_prefix(&self, p: K::SubPrefix) -> Prefix<T> {
         Prefix::new(self.pk_namespace, &p.prefix())
     }
+
+    // use no_prefix to scan -> range
+    fn no_prefix(&self, p: K::NoPrefix) -> Prefix<T> {
+        Prefix::new(self.pk_namespace, &p.prefix())
+    }
```

Comment:
I noticed this repeated implementation and started to wonder why it's not solved with traits.
I proposed some design here: https://github.com/CosmWasm/cw-plus/pull/436
It's neither finished nor production ready, but I wanted to make a point about functionality.
Please let me know what you think.

Reply:
As discussed below, it's not needed. I just got confused by the existing impl / usage of `EmptyPrefix`.